### PR TITLE
Adds a prototype  SCIM-Only-Okta-Integration install guide

### PR DESCRIPTION
**Pull request by tcsc**

### CODE REVIEW

File: docs/pages/application-access/okta/scim-only.mdx [hunk 1]
```diff
@@ -0,0 +1,166 @@
+---
+title: Installing SCIM-only Okta integration
+description: How to install a SCIM-only Okta integration
+---
+
+## Before you begin
+This test involves upgrading to a dev build of Teleport 15 - back up your
+cluster config & back-end data just in case.
```
Comment (r0mant) [inline docs/pages/application-access/okta/scim-only.mdx:8]: I would remove this, the goal for this is to just become a regular guide in the docs. I will share the dev build with interested customers as needed before this makes it into an actual release.

File: docs/pages/application-access/okta/scim-only.mdx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+<Admonition type="warning">
+Okta does not send SCIM updates to Teleport when a user is merely suspended. Even though Okta will prevent a suspended user from logging back into the cluster, any existing sessions will not be terminated, and any pre-issued credentials will be valid for their normal lifetimes.
+</Admonition>
+
+## What you need
+ * Enterprise Teleport Dev Build `v15.3.2-dev.scim.1`, available via 
+   * AMD64 linux: https://cdn.teleport.dev/teleport-ent-v15.3.2-dev.scim.1-linux-amd64-bin.tar.gz 
+   * Arm64 linux: https://cdn.teleport.dev/teleport-ent-v15.3.2-dev.scim.1-linux-arm64-bin.tar.gz
+   * Multiarch Docker image (with debug tools): public.ecr.aws/gravitational/teleport-ent-distroless-debug:15.3.2-dev.scim.1
```
Comment (r0mant) [inline docs/pages/application-access/okta/scim-only.mdx:45]: Similar to above, I would remove the mention of a dev build.

File: docs/pages/application-access/okta/scim-only.mdx [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+ * A Teleport cluster with a working SAML Connector to your Okta organization
+ * A securely-generated bearer token
+
+## Installing Teleport
+
+If installing from one of the above tarballs onto a self-hosted system, first 
+back up your existing binaries then unpack the linked tarball and move the 
+contained binaries into place.
```
Comment (r0mant) [inline docs/pages/application-access/okta/scim-only.mdx:53]:
```suggestion
```

File: docs/pages/application-access/okta/scim-only.mdx [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    cert: REDACTED
+    private_key:  REDACTED
+  sso: https://example.okta.com/app/….
+version: v2
+```
+
+## Installing the Teleport SCIM integration
+You can only install the SCIM integration with the build of `tctl` supplied with
+the `v15.3.2-dev.scim.1` build of Teleport. Other builds of `tctl` do not yet
+have the necessary commands.
```
Comment (r0mant) [inline docs/pages/application-access/okta/scim-only.mdx:95]:
```suggestion
Run the following `tctl` command to install the SCIM integration.
```

**Review by r0mant — COMMENTED** [review]
Let's just make this a normal guide without being dev-build specific, otherwise looks good.

File: docs/pages/application-access/okta/scim-only.mdx [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+## How it works
+User provisioning (and de-provisioning) with SCIM requires two Teleport
+components working together:
+
+ * A SAML Connector that provides SSO login to Teleport for upstream Okta users
+ * A SCIM integration that provisions and de-provisions Teleport user accounts
+   in response to changes in the upstream Okta organization
+
+Both of these Teleport components rely on an Okta SAML Application to act as the
+interface between Teleport and Okta on the Okta side. For consistency, both of
```
Comment (r0mant) [inline docs/pages/application-access/okta/scim-only.mdx:19]: "Okta on the Okta side" reads strange.
```suggestion
Both of these Teleport components rely on an Okta SAML Application to act as the
interface between Teleport and Okta. For consistency, both of 
```

File: docs/pages/application-access/okta/scim-only.mdx [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+## Installing Teleport
+
+If installing from one of the above tarballs onto a self-hosted system, first 
+back up your existing binaries then unpack the linked tarball and move the 
+contained binaries into place.   
+
+## Installing the SAML Connector
+If you created an Okta SAML connector with the Hosted Okta Integration enrollment
+flow, it will have created a SAML Connector called okta-integration that assigns
+users the okta-requester to the Okta group Everyone.
```
Comment (r0mant) [inline docs/pages/application-access/okta/scim-only.mdx:58]: This part is slightly confusing - if you have hosted Okta integration configured, you already have SCIM enabled so why would you be going through the standalone SCIM guide?

File: docs/pages/application-access/okta/scim-only.mdx [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+
+If you don’t already have a working SAML Connector, see the 
+[Teleport Authentication with Okta](../../access-controls/sso/okta) guide. You
+can skip the Okta Group and role creation parts, as they’re not necessary for a
+SCIM deployment
+
+The resulting Teleport SAML connector resource should look roughly like the 
+example below. Note the role mapping that grants the requester role to the
+`Everyone` group in Okta.
```
Comment (r0mant) [inline docs/pages/application-access/okta/scim-only.mdx:67]: This phrasing is slightly confusing - is it a requirement to have the role mapping for "everyone" setup, or just an example and users can use any existing connector they have?

File: docs/pages/application-access/okta/scim-only.mdx [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+</Admonition>
+
+In general, best practice is to assign a no-access role like `requester` as a
+default, and add more sophisticated controls via Access Lists.
+
+### Installation Notes:
+ * The `--token` parameter can also be read from the `TELEPORT_SCIM_BEARER_TOKEN`
+   environment variable to hide it from your shell history. 
+ * The `--token` value should just be the raw token value, with no Bearer prefix
+ * `--role` defaults to `requester` if not supplied
```
Comment (r0mant) [inline docs/pages/application-access/okta/scim-only.mdx:121]:
```suggestion
 * `--role` defaults to the built-in `requester` preset role if not supplied
```

File: docs/pages/application-access/okta/scim-only.mdx [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+Teleport, you can edit the Okta Application User profile to present Teleport 
+with a subset and/or mapped version of the full User profile. 
+
+## Deleting the SCIM integration
+
+You can delete the SCIM integration via the Integrations page in the Teleport UI,
+or with tctl like so:
+
+```bash
+tctl plugins delete okta-scim
```
Comment (r0mant) [inline docs/pages/application-access/okta/scim-only.mdx:150]:
```suggestion
$ tctl plugins delete okta-scim
```

File: docs/pages/application-access/okta/scim-only.mdx [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+version: v2
+```
+
+## Installing the Teleport SCIM integration
+You can only install the SCIM integration with the build of `tctl` supplied with
+the `v15.3.2-dev.scim.1` build of Teleport. Other builds of `tctl` do not yet
+have the necessary commands.
+
+```bash
+tctl plugins install scim \
```
Comment (r0mant) [inline docs/pages/application-access/okta/scim-only.mdx:98]:
```suggestion
$ tctl plugins install scim \
```

**Review by r0mant — COMMENTED** [review]
Left some comments after second pass.

**Comment by smallinsky** [comment]
Closing in favor of https://github.com/gravitational/teleport/pull/42072

**Pull request closed without merging**